packages/office-ui-fabric-react/src/components/Nav/Nav.scss

```diff
@@ -145,7 +145,7 @@ $linkRightPadding: 20px;
       background: $selectedBackgroundColor;
 
       &:after {
-        @include border-left(2px, solid, $selectedForegroundColor);
+        @include border-left(2px, solid, $ms-color-themePrimary);
```

Comment:
This was using a semantic color but now you're changing it to use a non semantic color. I don't think this is expected.